source/incidents-and-alerts/index.html.md.erb

```diff
@@ -155,3 +155,31 @@ Throughout the incident, record actions taken, decisions, significant external c
 ## After the incident
 
 When the incident is resolved, the incident lead needs to record the completed template with a unique ID in the [Incident Log](https://docs.google.com/spreadsheets/d/1NKk_XgXFooHSLlnKtIGMsWeKy9Ws3WU61VtcRdH2OIE/edit).
+
+## Helping other organisation incident management teams with their incidents
+
+From time to time GovWifi may be requested to assist an organisation that is in incident management mode. The following principles should help make the engagement easier.
+
+### Identify the admin user for the organisation
+
+Within the admin portal, locate the admin user listed for the organisation in question. The incident lead for the organisation is unlikely to be our admin user so confirm that the admin user is also available in any resolution call.
```

Comment:
Can you change this to 'Within GovWifi admin' (as set out in the style guide) so all our documentation stays consistent?

Reply:
Also, is the key point of this section that we want the admin user to be involved? 

I think this could be written clearer as it took me a couple of reads to be sure. 

Perhaps if we changed "so confirm that the admin user is also available in any resolution call" to "so invite the admin user to any resolution call" ? Just so the action for the team is clearer?

Also I might just be being picky here - but what if there are multiple admin users? Perhaps we should say 'at least one admin user' instead?

Reply:
Those fixes applied now... I adjusted it a little further around the "multiple admin users... i.e.


> In GovWifi admin, locate the admin users listed for the organisation in question. The incident lead for the organisation is unlikely to be an admin user so invite at least one admin user to any resolution call.